_episodes/11-using-arrays-transformations.md

```diff
@@ -21,31 +21,28 @@ Arrays can be sorted, de-duplicated, and manipulated in other ways in GREL expre
 
 "Monday,Tuesday,Wednesday,Thursday,Friday,Saturday,Sunday"
 
-This can be transformed into an array using the ```split``` function
+This can be transformed into an array using the ```split``` function with pipe
 ```
-value.split(",")
+value.split("|")
 ```
-This would create the array containing the days of the week:
+This would create the array containing a list of subject headings, separated by a pipe character | (as in the data)
```

Comment:
Rephrase to be clear that the original list is turned into an array based on using the pipe character as a separator. In the transformation preview the array will display as a list of comma separated values in double quotes, with the whole array surrounded by square brackets